package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jsdom",
-  "version": "11.2.0",
+  "version": "11.2.1",
```

Comment:
Please don't change the version of jsdom

Reply:
@domenic done